gcsweb.k8s.io/deployment.yaml

```diff
@@ -27,6 +27,7 @@ spec:
             - -b=kubernetes-jenkins
             - -b=kubernetes-release
             - -b=kubernetes-release-dev
+            - -b=android-ci
```

Comment:
nit: I think this list is alphabetized, can you please move it before `-b=crreleases`?

Reply:
Done